App/Components/Atom/Text/index.js

```diff
@@ -82,7 +82,7 @@ const Text = ({children, preset, style, ...rest}) => {
 }
 
 Text.propTypes = {
-  preset: PropTypes.oneOf(...Preset)
+  preset: PropTypes.oneOf(...Object.keys(Preset))
```

Comment:
The first error led me to here. `Preset` is an object. You can't spread and object however it seems you wanted to make sure that the prop was one of the _keys_ of the object. `Object.keys` returns an array of keys which you could spread. That said since `Object.keys` returns an array you could likely skip the spread in this case all together